testdata/files/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.json
```

Comment:
Why this file?

Reply:
I did some tests with json and forgot to remove it, should be gone now 👍